lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -16,7 +16,8 @@ class MiqGenericMountSession < MiqFileStorage::Interface
 
   class NoSuchFileOrDirectory < RuntimeError; end
 
-  attr_accessor :settings, :mnt_point, :logger
+  attr_accessor :settings, :mnt_point
+  attr_reader :logger
```

Comment:
Line 28 is the _getter_ not the setter.  Think you meant to make this `attr_writer`?